terraform/cloud-platform/templates/kops.yaml.tpl

```diff
@@ -170,29 +170,6 @@ spec:
   api:
     loadBalancer:
       type: Public
-  additionalPolicies:
-    node: |
-      [
-        {
-          "Effect": "Allow",
-          "Action": [
-            "route53:ChangeResourceRecordSets"
-          ],
-          "Resource": [
-            "arn:aws:route53:::hostedzone/${hosted_zone_id}"
-          ]
-        },
-        {
-          "Effect": "Allow",
-          "Action": [
-            "route53:ListHostedZones",
-            "route53:ListResourceRecordSets"
-          ],
-          "Resource": [
-            "*"
-          ]
-        }
-      ]
```

Comment:
Built a test cluster using this branch and the above changes. Unfortunately, `kops` doesn't ever validate. 
I just end up with:
```
Using cluster from kubectl context: cloud-platform-kiam-test.k8s.integration.dsd.io

Validating cluster cloud-platform-kiam-test.k8s.integration.dsd.io

unexpected error during validation: unable to resolve Kubernetes cluster API URL dns: lookup api.cloud-platform-kiam-test.k8s.integration.dsd.io on xxx.xxx.x.x(redacted):53: no such host
```

Reply:
I may have been a little haste. :)

This did eventually validate. Nothing to see here.